### PR TITLE
Feature/mark interest frontend

### PR DESCRIPTION
**Pull request by eliasvagan**
Still needs API calling for liking metadata, but waiting for that to be completed by @petterroea .

### CODE REVIEW

File: frontend/src/pages/Inspection/MetaData.jsx [hunk 1]
```diff
@@ -1,92 +1,228 @@
...
+      const userSelector = useSelector((state) => state.user);
+      const { token } = userSelector.user;
+      await fetch(`/api/SOMETHING-FOR-ADDING-LIKE/${uuid}/${token}`);
+    } catch (err) {
+      dispatch(alertActions.error('Something went wrong when adding/removing favourite.'));
+    }
+  };
 
   return (
     <Wrapper>
```
Comment (AndreasJJ) [inline frontend/src/pages/Inspection/MetaData.jsx]: this should probably be two components "MetadataContent" and "MetadataToolbar".

File: frontend/src/pages/Inspection/MetaData.jsx [hunk 2]
```diff
@@ -1,92 +1,228 @@
...
+      </MetadataContent>
+      <MetadataToolbar>
+        <FavouriteButton onClick={handleLike}>
+          {isLiked ? <StarFullStyled /> : <StarEmptyStyled />}
+          <p>Interesting</p>
+          <LikeCounter>{likes}</LikeCounter>
+        </FavouriteButton>
+        <div>
+          {
+          experiencePostGuid !== undefined
```
Comment (AndreasJJ) [inline frontend/src/pages/Inspection/MetaData.jsx]: There's no reason to have " experiencePostGuid !== undefined".  "!experiencePostGuid" works fine as null or undefined will be equal to false. Not to mention that this doesn't work when it's null.

Skjermbilde 2020-02-29 kl  12 32 26

File: frontend/src/pages/MetadataByMunicipality/FeedbackLabel.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import negativeIcon from '../../assets/ui/negative.svg';
+import positiveIcon from '../../assets/ui/positive.svg';
```
Comment (AndreasJJ) [inline frontend/src/pages/MetadataByMunicipality/FeedbackLabel.jsx]: Tfw you don't use styled-icons

File: frontend/src/pages/Inspection/MetaData.jsx (hunk 1 above)
Comment (eliasvagan) [reply frontend/src/pages/Inspection/MetaData.jsx]: Working on this one.

**Comment by eliasvagan** [comment]
Still waiting for API for PUT-ing likes, but it's getting there.

File: frontend/src/pages/Inspection/MetaData.jsx (hunk 1 above)
Comment (eliasvagan) [reply frontend/src/pages/Inspection/MetaData.jsx]: Resolved in new commit now.

**Pull request merged**